### ISSUE TITLE
sv_maxclients limit to change to 64

### ISSUE THREAD

**Issue by AlexCSilva** [JKGDevs/JediKnightGalaxies#196]
Use to work on regular jamp, be useful to have in the future.

**Comment by AlexCSilva** [comment]
Would it be in server or game

**Comment by ensiform** [comment]
Its a requirement everywhere not just `server` or `game`.

**Comment by AlexCSilva** [comment]
oic never knew about that

**Comment by DarthFutuza** [comment]
We could maybe just increase the entity total count by 32, but that may affect performance significantly.

**Comment by eezstreet** [comment]
I'm starting to have second thoughts about this.
On the one hand, it's a nice feature to have for right now.
On the other hand, doing this means that increasing the limit to be an arbitrary amount is going to totally undo the work here, so I'm not sure it's worth it. Plus we don't have the player count really to warrant it yet.
I don't know, maybe other people can chime in?

**Comment by ensiform** [comment]
The way I look at it is, you will have to then come up with raising entity limits too because: technically clients use up max_clients*2 entity slots for the client list + the bodyque.  Plus you need to ensure adding more to max_clients wont overflow the CS_MAX (it should error the compile if it does)

**Comment by DarthFutuza** [comment]
Dynamically allocated entity limits? (With an arbitrary limit for safety purposes?)

**Comment by ensiform** [comment]
I don't think that will work very well given so many loops all over the place require a hard limit.
